packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -210,6 +210,7 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
     ): Promise<IDocumentService> {
         ensureFluidResolvedUrl(resolvedUrl);
 
+        let ordererRestWrapper: RouterliciousOrdererRestWrapper | undefined;
```

Comment:
Since we know that we want an OrdererRestWrapper and/or its token somewhere down the line (either during discovery or shortly thereafter with getDeltas or for retrieving a token to connect to deltaStream), I think it would make more sense here to move the load() from 227 to here, and make this a const that cannot be undefined. Then you can remove the undefined checks for the param in DocService, as well.